scripts/build.sh

```diff
@@ -36,16 +32,9 @@ if [ -z "${SKIP_CARGO_BUILD:-}" ]; then
 fi
 export PATH="$PWD/target/${CARGO_PROFILE:-debug}:$PATH"
 
-
-# Function for killing processes stored in FM_PID_FILE in reverse-order they were created in
-function kill_fedimint_processes {
-  echo "Killing fedimint processes"
-  PIDS=$(cat $FM_PID_FILE | sed '1!G;h;$!d') # sed reverses order
-  if [ -n "$PIDS" ]
-  then
-    kill $PIDS 2>/dev/null
-  fi
-  rm -f $FM_PID_FILE
+function kill_devimint {
+  echo "Killing devimint and child processes"
+  pkill devimint
```

Comment:
Does this need -9?

Reply:
no I think, SIGTERM is better. this will allow devimint to kill its children.

Reply:
It's probably this (causing tests to fail). It's just very meh way to do stuff.